src/main/java/fr/openent/viescolaire/controller/TimeSlotController.java

```diff
@@ -140,8 +139,13 @@ public void getDefaultStructureTimeSlot(final HttpServerRequest request) {
         });
     }
 
+    /**
+     * @param request
+     * @queryParam {structureId} mandatory
+     */
     @Put("/time-slots")
```

Comment:
A voir pour changé en 
```suggestion
    @Put("/structure/:structureId/time-slots")
```
pour respecter le REST API.
Je pense que tu peux faire cette modification puis que de tout façon tu modifie tout les appels a cause du @queryParam {structureId} mandatory

@lapeyronnie qu'est ce que tu en pense ?

Reply:
En l'occurrence pour cet endpoint, le body était déjà composé de la structureId avant les modifs.
A voir du coup s'il est judicieux de faire cette modif.